root/components/home/Explore.js

```diff
@@ -0,0 +1,186 @@
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Community Driven')}
+                </a>
+              </h5>
+              <p className="card-text">
+                {l(`MusicBrainz is a community-maintained
+                  open source encyclopedia of music information.
+                  This means that anyone — including you — can
+                  help contribute to the project by adding
+                  information about your favorite artists
+                  and their works.`)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card">
+            <div className="card-body">
+              <h5 className="card-title">
+                <a
+                  href="https://musicbrainz.org/doc/Development"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Development')}
+                </a>
+              </h5>
+              <p className="card-text">
+                {l(`If you have a digital music collection,
+                  MusicBrainz Picard will help you tag your files.
+                  If you are a developer, our developer
+                  resources will help you in making
+                  use of our data. If you are a
+                  commercial user, our live data feed will provide your
+                  local database with replication
+                  packets to keep it in sync.`)}
+              </p>
+            </div>
+          </div>
+
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card">
+            <div className="card-body">
+              <h5 className="card-title">
+                <a
+                  href="https://musicbrainz.org/doc/General_FAQ#Why_would_I_need_to_use_MusicBrainz.3F_What.27s_wrong_with_Gracenote.27s_CDDB.3F"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('History')}
+                </a>
+              </h5>
+              <p className="card-text">
+                {l(`In 2000, Gracenote took over the free
+                  CDDB project and commercialized it,
+                  essentially charging users for accessing
+                  the very data they themselves contributed.
+                  In response, Robert Kaye founded MusicBrainz.
+                  The project has since grown rapidly from
+                  a one-man operation to an international
+                  community of enthusiasts that appreciates
+                  both music and music metadata. Along the way,
+                  the scope of the project has expanded from
+                  its origins as a mere CDDB replacement
+                  to the true music encyclopedia MusicBrainz is today.`)}
+              </p>
+            </div>
```

Comment:
TBH this seems such ancient history (and kinda revenge-seeking) that I'd drop any references to Gracenote and just try to have a positive, "we are good because of this" vibe that doesn't speak of others.

Reply:
Makes sense. Fixed